tokio-quiche/src/metrics/labels.rs

```diff
@@ -105,6 +105,7 @@ impl From<&BoxError> for HandshakeError {
 pub enum QuicInvalidInitialPacketError {
     TokenValidationFail,
     SmallPacket,
+    FailedToParse,
```

Comment:
One minor thing to discuss: should I merge `SmallPacket` and `FailedToParse` into one error kind? I don't think anyone cares about the distinction between those 2 cases.

Reply:
That's fine by me